mlflow/models/container/__init__.py

```diff
@@ -105,7 +105,7 @@ def _install_pyfunc_deps(
         conf = model.flavors[pyfunc.FLAVOR_NAME]
```

Comment:
I see that `env_manager=em.CONDA` is still the default for `_install_pyfunc_deps`. Shall we make it virtualenv?

Reply:
Thanks for the catch! Yep, we should. I will.

Reply:
@dbczumar Filed https://github.com/mlflow/mlflow/pull/6719.